### DISCUSSION TITLE
Notification

### DISCUSSION THREAD

**Discussion by sjuergen** [simatic-ax/.discussions#2] [Q&A] [answered]
who will be notificated, when a new questions will asked?

**Comment by RauerJ** [comment, marked as answer]
Everybody who has subscribed to the activity of https://github.com/simatic-ax/.discussions will be informed when a new question is asked. Just click on the link and select Watch -> "All activity" on the right hand side (see screenshot).

If a question you asked is answered by someone else, you are also informed if you haven't subscribed to all activity.

![Discussions_Subscribe](https://user-images.githubusercontent.com/95907495/175253673-2015b876-11f2-4f3d-86b1-405c8a83635d.png)

**Reply by sjuergen** [reply]
nice! But can I receive also a notification on new questions?

**Reply by theBadT** [reply]
![grafik](https://user-images.githubusercontent.com/1228538/175261786-26254002-329e-43d6-b049-05fe5578de77.png)
